src/main/zapHomeFiles/hud/libs/i18n.js

```diff
@@ -0,0 +1,256 @@
+      alerts_details_title: "Alert Details",
+      alerts_field_attack: "Attack",
+      alerts_field_confidence: "Confidence",
+      alerts_field_cweid: "CWE Id",
+      alerts_field_description: "Description",
+      alerts_field_evidence: "Evidence",
+      alerts_field_other: "Other Info",
+      alerts_field_parameter: "Parameter",
+      alerts_field_reference: "Reference",
+      alerts_field_risk: "Risk",
+      alerts_field_solution: "Solution",
+      alerts_field_url: "URL",
+      alerts_field_wascid: "WASC Id",
+      alerts_page_high_tool: "Page Alerts: High",
+      alerts_page_info_tool: "Page Alerts: Info",
+      alerts_page_low_tool: "Page Alerts: Low",
+      alerts_page_medium_tool: "Page Alerts: Medium",
+      alerts_page_title: "Page Alerts",
+      alerts_risk_high: "High",
+      alerts_risk_info: "Informational",
+      alerts_risk_low: "Low",
+      alerts_risk_medium: "Medium",
+      alerts_site_high_tool: "Site Alerts: High",
+      alerts_site_info_tool: "Site Alerts: Info",
+      alerts_site_low_tool: "Site Alerts: Low",
+      alerts_site_medium_tool: "Site Alerts: Medium	",
+      alerts_site_title: "Site Alerts",
+      alerts_title: "Alerts",
+      api_error_title: "API Error",
+      ascan_start_1: "Start Active Scan on <i>",
+      ascan_start_2: "</i>?",
+      ascan_start_scope_1: "<i>",
+      ascan_start_scope_2:
+        "</i> is not in scope.<br>In order to Active Scan the site you must add it to the scope.<br>Add <i>",
+      ascan_start_scope_3: "</i> to the scope and start Active Scanning it?",
+      ascan_stop_1: "The site <i>",
+      ascan_stop_2:
+        "</i> is currently getting Active Scanned. <br> Would you like to stop the Active Scan tool?",
+      ascan_tool: "Active Scan",
+      attack_start: "Turn off Attack Mode?",
+      attack_stop:
+        "Turn on Attack Mode?<br>This will cause ZAP to automatically attack all pages in scope.",
+      attack_tool: "Attack Mode",
+      break_continue: "Continue",
+      break_drop: "Drop",
+      break_http_message_title: "HTTP Message",
+      break_intercept_title: "Intercepted HTTP Message",
+      break_step: "Step",
+      break_tool: "Break",
+      common_add: "Add",
+      common_cancel: "Cancel",
+      common_clear: "Clear",
+      common_enable_regex: "Enable RegEx",
+      common_in: "In",
+      common_items_filtered: "{0} of {1} items hidden by filters",
+      common_menu_title: "Menu",
+      common_off: "Off",
+      common_ok: "OK",
+      common_on: "On",
+      common_out: "Out",
+      common_remove: "Remove",
+      common_request: "Request",
+      common_response: "Response",
+      common_save: "Save",
+      common_start: "Start",
+      common_stop: "Stop",
+      common_turn_off: "Turn Off",
+      common_turn_on: "Turn On",
+      error_invalid_html_header: "Invalid HTML header",
+      error_with_message:
+        "Error: {0}.<br>See console log and zap.log for more details.",
```

Comment:
This is an example of HTML in the object. Just don't.

Reply:
@nothingismagick: Is that a complaint about the `<br>`?